presto-docs/src/main/sphinx/release/release-0.205.rst

```diff
@@ -0,0 +1,86 @@
+=============
+Release 0.205
+=============
+
+General Changes
+---------------
+
+* Fix parsing of row types where the field types contain spaces.
+  Previously, row expressions that included spaces would fail to parse.
+  For example: ``cast(row(timestamp '2018-06-01') AS row(timestamp with time zone))``.
+* Fix distributed planning failure for complex queries when using bucketed execution.
+* Fix :func:`ST_ExteriorRing` to only accept polygons.
+  Previously, it erroneously accepted other geometries.
+* Add the ``task.min-drivers-per-task`` and ``task.max-drivers-per-task`` config options.
+  The former specifies the guaranteed minimum number of drivers a task will run concurrently
+  given that it has enough work to do. The latter specifies the maximum number of drivers
+  a task can run concurrently.
+* Add the ``concurrent-lifespans-per-task`` config property to control the default value
+  of the ``concurrent_lifespans_per_task`` session property.
+* Add the ``query_max_total_memory`` session property and the ``query.max-total-memory``
+  config property. Queries will be aborted after their total (user + system) memory
+  reservation exceeds this threshold.
+* Improve stats calculation for outer joins and correlated subqueries.
+* Reduce memory usage when a ``Block`` contains all null or all non-null values.
+* Change the internal hash function used in  ``approx_distinct``. The result of ``approx_distinct``
+  may change in this version compared to the previous version for the same set of values. However,
+  the standard error of the results should still be within the configured bounds.
+* Improve efficiency and reduce memory usage for scalar correlated subqueries with aggregations.
+* Remove the legacy local scheduler and associated configuration properties,
+  ``task.legacy-scheduling-behavior`` and ``task.level-absolute-priority``.
+* Do not allow using the ``FILTER`` clause for the ``COALESCE``, ``IF``, or ``NULLIF`` functions.
+  The syntax was previously allowed but was otherwise ignored.
+
+Security Changes
+----------------
+
+* Remove unnecessary check for ``SELECT`` privileges for ``DELETE`` queries.
+  Previously, ``DELETE`` queries could fail if the user only has ``DELETE``
+  privileges but not ``SELECT`` privileges.
+  This only affected connectors that implement ``checkCanSelectFromColumns()``.
+* Add a check that the view owner has permission to create the view when
```

Comment:
This only affects connectors that implement `checkCanCreateViewWithSelectFromColumns()`.   (we were doing the right thing for `checkCanCreateViewWithSelectFromTable()`)